alacritty/src/display/mod.rs

```diff
@@ -825,3 +834,263 @@ fn window_size(
 
     PhysicalSize::new(width as u32, height as u32)
 }
+
+/// Keyboard regex hint state.
+pub struct HintState {
```

Comment:
Maybe we should move this `Hint` stuff to a separate `mod` instead of putting right into `mod.rs` (with test ofc)? Like `hint.rs` should work for it, I think.

Reply:
It would work, yeah. I'll consider it again. Though it is somewhat tiny.

Reply:
With 260 LoC this is actually about the ideal size for a new module, I didn't think it was this big. Good suggestion.